addons/product/tests/test_product_pricelist.py

```diff
@@ -85,17 +85,11 @@ def setUp(self):
         })
 
         self.uom_unit_id = self.ref('uom.product_uom_unit')
-        self.list0 = self.ref('product.list0')
 
         self.ipad_retina_display.write({'uom_id': self.uom_unit_id, 'categ_id': self.category_5_id})
         self.customer_pricelist = self.ProductPricelist.create({
             'name': 'Customer Pricelist',
             'item_ids': [(0, 0, {
-                'name': 'Default pricelist',
-                'compute_price': 'formula',
-                'base': 'pricelist',
-                'base_pricelist_id': self.list0
-            }), (0, 0, {
```

Comment:
This logic should be kept imho, even if the price is the same, it tested the pricelist base logic.
You should create a new dummy pricelist instead of the list0 one.